manifests/k/kefir500/apk-editor-studio/1.6.0/kefir500.apk-editor-studio.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: kefir500.apk-editor-studio
+PackageVersion: 1.6.0
+PackageLocale: en-US
+Publisher: kefir500

```

```suggestion
Publisher: Alexander Gorishnyak
```
This must match Control Panel.